pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -968,3 +974,45 @@ func (h *upgradeHandler) startVM(ctx context.Context, vm *kubevirtv1.VirtualMach
 		Do(ctx)
 	return res.Error()
 }
+
+// checkLogReadyCondition times out LogReady condition, thus moving on to the next step if upgradeLogCondLogReadyTimeout
+// time has passed since UpgradeLog's logging infrastructure setup started
+func (h *upgradeHandler) checkLogReadyCondition(upgrade *harvesterv1.Upgrade) (*harvesterv1.Upgrade, error) {
+	upgradeConfig, err := settings.DecodeConfig[settings.UpgradeConfig](settings.UpgradeConfigSet.Get())
+	if err != nil {
+		return upgrade, err
```

Comment:
log an error, and use the default timeout value to continue

Reply:
as a minor enhancement, would prefer to block the upgrade main process as less as possbile;
it can return error and wait user to sovle it; or use the default value to continue;   the (default value) 5 mintues looks to be enough for most cases to get either success or failure result

Reply:
Done. Again, can you PTAL if I understood you correctly?